shared/chat/conversation/messages/message-popup/text/index.js

```diff
@@ -37,7 +37,7 @@ const TextPopupMenu = (props: Props) => {
             danger: true,
             disabled: !props.onDelete,
             onClick: props.onDelete,
-            subTitle: 'Deletes this message for everyone',
+            subTitle: 'Deletes this message for everyone.',
```

Comment:
🐑 this isn't a complete sentence, so it doesn't need a period

Reply:
Lol, fair enough. `Deletes this message for everyone` is a fragment, though. Definitely not a complete sentence (<-- no period)

Reply:
Alright alright - fixed.

Reply:
🙇 🙇 🙇